core/tests/test_apps.cardano.address.py

```diff
@@ -431,13 +451,26 @@ def test_shelley_address_with_byron_namespace(self):
         keychain = Keychain([44 | HARDENED, 1815 | HARDENED], node)
 
         with self.assertRaises(wire.DataError):
-            derive_base_address(keychain, [44 | HARDENED, 1815 | HARDENED, HARDENED, 0, 0], 0, None)
+            address_parameters = CardanoAddressParametersType(
+                address_type=CardanoAddressType.BASE_ADDRESS,
+                address_n=[1852 | HARDENED, 1815 | HARDENED, 0 | HARDENED, 0, 0]
```

Comment:
I just noticed that I have the wrong namespace here - it should be 44 | HARDENED. I'll fix it later.